### PR TITLE
chore: Add verbose logs for periodic commands

### PR DESCRIPTION
**Pull request by ItayLevyOfficial**
# PR Standards

## Opening a pull request should be able to meet the following requirements

---

For Author:

- [x]  Targeted PR against correct branch
- [x]  included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [x]  Linked to Github issue with discussion and accepted design
- [x]  Targets only one github issue
- [ ]  Wrote unit and integration tests
- [ ]  All CI checks have passed
- [ ]  Added relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code)

---

For Reviewer:

- [ ]  confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ]  Reviewers assigned
- [ ]  confirmed all author checklist items have been addressed

---

After reviewer approval:

- [ ]  In case targets main branch, PR should be squashed and merged.
- [ ]  In case PR targets a release branch, PR should be rebased.

### CODE REVIEW

**Comment by mtsitrin** [comment]
if I remember correctly it was spamming the relayer log

File: cmd/utils/bash_commands.go [hunk 1]
```diff
@@ -20,9 +20,9 @@ func RunCommandEvery(ctx context.Context, command string, args []string, interva
 			for _, option := range options {
 				option(cmd)
 			}
-			err := cmd.Run()
+			_, err := ExecBashCommandWithStdout(cmd)
 			if err != nil {
-				cmd.Stderr.Write([]byte(fmt.Sprintf("Failed to execute command: %s, err: %s\n", command, err)))
+				cmd.Stderr.Write([]byte(fmt.Sprintf("Failed to execute command: %s, err: %s\n", cmd, err)))
```
Comment (mtsitrin) [inline cmd/utils/bash_commands.go:25]: what the `CommandContext` return when u `%s` printf it?

Comment (ItayLevyOfficial) [reply cmd/utils/bash_commands.go]: good catch!

**Comment by ItayLevyOfficial** [comment]
> if I remember correctly it was spamming the relayer log

IDK about that, but those error logs (from the [discord](https://discord.com/channels/956961633165529098/1158380243816550400/1158648840602271794)) with only the `exit status 1` are not saying much and it's hard to understand the problem from them.

**Review by omritoptix — APPROVED** [review]

**Pull request merged**